README.md

```diff
@@ -9,10 +9,12 @@ The code here has also been ported to:
 - [C++][cpp] and [Rust][rust] by Jürgen Wurzer,
 - [Python][python] by Kuba Sunderland-Ober.
 - [MiniScript][miniscript] by Joe Strout.
+- [TypeScript][typescript] by May Tusek. <!-- Not sure if this should be a comma or period, IMO refactor all to periods. -->
```

Comment:
If you'd like, probably the best answer is no punctuation at all. If you'd like to do that clean-up, I'd appreciate it. If not, that's OK. :)

Reply:
Punctuation has been removed :)

Reply:
Thank you!